echo-core/src/main/java/com/netflix/spinnaker/echo/services/Front50Service.java

```diff
@@ -15,6 +15,10 @@
   @Headers("Accept: application/json")
   List<Pipeline> getPipelines(@Path("application") String application);
 
+  // either an empty list or a singleton of a raw pipeline config
```

Comment:
Under what conditions is a pipeline config returned here? Seems to me like we'd always be making live calls instead of looking things up in the cache.

Reply:
I think this always returns a pipeline config. We get a 404 if the pipeline id does not match anything.

Reply:
Is the pipeline config cache being used at all then if we're live calling first?

Reply:
yes, the pipeline config cache is used first and we are live calling last (just when something is about to be triggered)

Reply:
I missed the first step in my readthrough, makes sense now :+1: